Source/KakapoServer.swift

```diff
@@ -53,14 +53,30 @@ extension NSURLRequest {
     }
 }
 
-public class KakapoServer: NSURLProtocol {
-    
-    public typealias RouteHandler = Request -> Serializable?
+public typealias RouteHandler = Request -> Serializable?
+
+public struct Request {
+    let info: URLInfo
+    let HTTPBody: NSData?
+}
+
+public struct Response: CustomSerializable {
+    let code: Int
+    let headerFields: [String : String]?
+    let body: Serializable
+    
+    init(code: Int, headerFields: [String : String]? = nil, body: Serializable) {
```

Comment:
documentation say that optional params should be last (works anyway but is easier to understand)